packages/fluentui/accessibility/src/behaviors/Datepicker/datepickerCalendarCellBehavior.ts

```diff
@@ -5,14 +5,19 @@ import { Accessibility } from '../../types';
  * Behavior for a datepicker calendar grid.
  * @specification
  * Adds attribute 'aria-selected' based on the property 'selected' to 'root' slot.
+ * Adds role='gridcell'.
```

Comment:
Please could you add a specification for role on tableCell? And I think the comment for `gridcell` role also needs to have the slot specified.